provision/acc_provision/acc_provision.py

```diff
@@ -208,6 +208,9 @@ def config_default():
         "service_mesh_config": {
             "enable": False,
         },
+        "metallb_config": {
```

Comment:
As discussed, rename to load_balancer_config: {
"enable": False,
"lb_type": "metallb",
}